theme_odoo_experts/views/snippets/s_references.xml

```diff
@@ -4,7 +4,7 @@
 <template id="s_references" inherit_id="website.s_references">
     <!-- Section -->
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="pt0 pb56 o_cc o_cc2" remove="pt48 pb32" separator=" "/>
+        <attribute name="class" add="pt0 pb56 o_cc2" remove="pt80 pb80 o_cc1" separator=" "/>
```

Comment:
I think you should also remove the link :)
![Capture d’écran 2024-08-01 à 14 37 19](https://github.com/user-attachments/assets/a0d1eb54-bbd0-4288-8c42-0521f8b8f7c2)